test/methods.test.js

```diff
@@ -0,0 +1,141 @@
+const test = require('ava')
+const nock = require('nock')
+const f11s = require('data.js')
+const fs = require('fs')
+
+const { Client } = require('../lib/index')
+
+/**
+ * Push stuff
+ */
+const config = {
+  api: 'http://localhost:5000',
+  authToken: 'be270cae-1c77-4853-b8c1-30b6cf5e9878',
+  organizationId: 'myorg',
+  datasetId: 'my_dataset',
+}
+
+const client = new Client(
+  config.authToken,
+  config.organizationId,
+  config.datasetId,
+  config.api
+)
+
+test('push a metadata', async (t) => {
```

Comment:
If we wanted to get fancy we should have a test where we push a dataset with existing name and get a 409 conflict etc but that's minor.

Reply:
yea thought about this too, we can do this later